tests/test_automatic_payout_transactions.py

```diff
@@ -0,0 +1,96 @@
+from utils import stripe_obj_to_dict, client, midnight
+from tap_tester import runner, connections
+from base import BaseTapTest
+
+def get_payouts():
+    """
+        Return all the payouts (with pagination), to determine the automatic and non-automatic payouts
+    """
+    # list of all data to return
+    data = []
+    # api call of 1st page
+    stripe_obj = client["payouts"].list(limit=100, created={"gte": midnight})
+    dict_obj = stripe_obj_to_dict(stripe_obj)
+    # add data
+    data += dict_obj['data']
```

Comment:
Handle the exception where dict_obj can be empty if this test run without running all fields test. 

e.g. my test run failed throwing following exception.

```
======================================================================
ERROR: setUpClass (test_automatic_payout_transactions.AutomaticPayoutTransactionTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/code/tap-stripe/tests/test_automatic_payout_transactions.py", line 44, in setUpClass
    payouts = get_payouts()
  File "/opt/code/tap-stripe/tests/test_automatic_payout_transactions.py", line 17, in get_payouts
    data += dict_obj['data']
KeyError: 'data'
```

Reply:
When all fields test is not run below line returns the null

`dict_obj = stripe_obj_to_dict(stripe_obj)
   data += dict_obj['data']`

So can we please have try-catch around it and throw exception with appropriate message?